app/src/main/java/com/asynclabs/asyncsport/BlankFragment.kt

```diff
@@ -0,0 +1,59 @@
+package com.asynclabs.asyncsport
```

Comment:
THis should not be part of the task. Remove it.

Reply:
Si it is the user profile you created i just merged mine with paymentFragment

Reply:
FOllow the following to undo irrelevant changes to your pr and redo the pr . Delete this PR then redo it again. 

[Loom Link](https://www.loom.com/share/c2be5794f3b348fda41829d65fdc5b7c)